diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index 6b73a69..5a64087 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -111,9 +111,7 @@ def getReanimatedStaticFeatureFlags() {
     return featureFlags.collect { key, value -> "[${key}:${value}]" }.join("")
 }
 
-if (isNewArchitectureEnabled()) {
-    apply plugin: "com.facebook.react"
-}
+apply plugin: "com.facebook.react"
 
 def packageDir = project.projectDir.parentFile
 def reactNativeRootDir = resolveReactNativeDirectory()
@@ -292,14 +290,14 @@ task assertMinimalReactNativeVersionTask {
 
 preBuild.dependsOn(assertMinimalReactNativeVersionTask)
 
-task assertNewArchitectureEnabledTask {
-    onlyIf { !IS_NEW_ARCHITECTURE_ENABLED }
-    doFirst {
-        throw new GradleException("[Reanimated] Reanimated requires new architecture to be enabled. Please enable it by setting `newArchEnabled` to `true` in `gradle.properties`.")
-    }
-}
-
-preBuild.dependsOn(assertNewArchitectureEnabledTask)
+// task assertNewArchitectureEnabledTask {
+//     onlyIf { !IS_NEW_ARCHITECTURE_ENABLED }
+//     doFirst {
+//         throw new GradleException("[Reanimated] Reanimated requires new architecture to be enabled. Please enable it by setting `newArchEnabled` to `true` in `gradle.properties`.")
+//     }
+// }
+// 
+// preBuild.dependsOn(assertNewArchitectureEnabledTask)
 
 def validateWorkletsBuildResult = providers.exec {
     workingDir(projectDir.path)
