diff --git a/node_modules/react-native-worklets/android/build.gradle b/node_modules/react-native-worklets/android/build.gradle
index 021f5bc..f67f954 100644
--- a/node_modules/react-native-worklets/android/build.gradle
+++ b/node_modules/react-native-worklets/android/build.gradle
@@ -89,9 +89,7 @@ def getStaticFeatureFlags() {
     return featureFlags.collect { key, value -> "[${key}:${value}]" }.join("")
 }
 
-if (isNewArchitectureEnabled()) {
-    apply plugin: "com.facebook.react"
-}
+apply plugin: "com.facebook.react"
 
 def packageDir = project.projectDir.parentFile
 def reactNativeRootDir = resolveReactNativeDirectory()
@@ -308,14 +306,14 @@ task assertMinimalReactNativeVersionTask {
 
 preBuild.dependsOn(assertMinimalReactNativeVersionTask)
 
-task assertNewArchitectureEnabledTask {
-    onlyIf { !IS_NEW_ARCHITECTURE_ENABLED }
-    doFirst {
-        throw new GradleException("[Worklets] Worklets require new architecture to be enabled. Please enable it by setting `newArchEnabled` to `true` in `gradle.properties`.")
-    }
-}
-
-preBuild.dependsOn(assertNewArchitectureEnabledTask)
+// task assertNewArchitectureEnabledTask {
+//     onlyIf { !IS_NEW_ARCHITECTURE_ENABLED }
+//     doFirst {
+//         throw new GradleException("[Worklets] Worklets require new architecture to be enabled. Please enable it by setting `newArchEnabled` to `true` in `gradle.properties`.")
+//     }
+// }
+// 
+// preBuild.dependsOn(assertNewArchitectureEnabledTask)
 
 task prepareWorkletsHeadersForPrefabs(type: Copy) {
     from("$projectDir/src/main/cpp")
